pandas/tests/io/json/test_normalize.py

```diff
@@ -663,6 +676,22 @@ def test_missing_nested_meta(self):
                 errors="raise",
             )
 
+    def test_missing_nested_meta_traverse_empty_list_errors_ignore(self):
+        # If errors="ignore" and nested metadata is nullable, return nan
+        data = {"meta": "foo", "nested_meta": [], "value": [{"rec": 1}, {"rec": 2}]}
```

Comment:
```suggestion
        data = {"meta": "foo", "nested_meta": {}, "value": [{"rec": 1}, {"rec": 2}]}
```

I have lost track a bit on what we are trying to accomplish but this doesn't seem right to ignore an error when the meta argument starts peering into in an array instead of an object. Is this a TypeError on main or a KeyError? I feel like we are mixing the two up when we should not be